modules/fastcv/src/utils.cpp

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "precomp.hpp"
+
+namespace cv {
+namespace fastcv {
+
+bool isInitialized = false;
```

Comment:
The same initialization issue as in main repo.

Reply:
will add singleton similar to main